### DISCUSSION TITLE
need help re-mapping 34 key

### DISCUSSION THREAD

**Discussion by stylustrader** [manna-harbour/miryoku#320] [Q&A] [answered]
how can I remove the function layer (red) and then replace the right side super key with the function button?
this is for 34-key zmk wireless.

I'm kinda lost with the docs:

```
#define U_FUN U_NUM

```

**Comment by manna-harbour** [comment, marked as answer]
34-key boards have thumb combos enabled automatically, so you can still use all functions that require the tertiary thumb keys. See https://github.com/manna-harbour/miryoku_zmk#34ferris and https://github.com/manna-harbour/miryoku_zmk#key-emulation-combos.

If you want to replace Super with Fun for the right hand pinkie column home row key hold on Base, you'd substitute a custom Base layer (https://github.com/manna-harbour/miryoku/discussions/85) and replace `U_MT(LGUI, SQT)` with `U_LT(U_FUN, SQT)` on the line corresponding to https://github.com/manna-harbour/miryoku_zmk/blob/e6683e9f8b6c199b339208b1b501e88a7308ed48/miryoku/miryoku_babel/miryoku_layer_alternatives.h#L120.

**Reply by stylustrader** [reply]
upon further testing, I realize I can't use U_FUN

I need an alternative mod key (not a layer key) to replace the right hand pinkie Super key.

use case:
android has hardcoded physical shortcuts. I can't change them. Already has predefined cmd+shortcuts. I need to change the super key on the right side (which I use the most often) into another mod key and then I can simply change my i3 config using that new mod key.

how can I change to meta, hyper, or altgr ,or  something else?

**Reply by manna-harbour** [reply]
You can substitute custom layers with https://github.com/manna-harbour/miryoku/discussions/85. For modifiers see https://zmk.dev/docs/codes/modifiers#modifiers.
